openprescribing/frontend/views/bookmark_utils.py

```diff
@@ -274,7 +274,7 @@ def _best_or_worst_performing_in_period(self, period, best_or_worst=None):
         worst = []
         measure_filter = {
             "month__gte": self.months_ago(period),
-            "measure__tags__contains": ["core"],
+            "measure__include_in_alerts": True,
```

Comment:
This needs to be added at https://github.com/ebmdatalab/openprescribing/pull/2128/commits/4599722d046032d35b72b2c4d0714fcf244bf59e#diff-22c4b84611cefe89e70c6debe46babd7R339 too (should prob grep for `tag.*core`)?

Reply:
Oh wow, how did I miss those?!